crates/rpc/rpc/src/trace.rs

```diff
@@ -325,7 +325,9 @@ where
         // apply after and count to traces if specified, this allows for a pagination style.
         // only consider traces after
         if let Some(after) = after.map(|a| a as usize).filter(|a| *a < all_traces.len()) {
-            all_traces = all_traces.split_off(after);
+            all_traces.drain(0..after);
+        } else {
+            return Ok(vec![])
```

Comment:
this now returns an empty vec if after is missing from the request?

this doesn't sound right?

could we also explain what `after` does exactly, not immediately obvious after reading the docs here

Reply:
sorry, forgot it.

Reply:
I'll need to read the og parity's codebase to see what's the usage of this field :)

Reply:
lets follow erigon's impl here because the parity is no longer in use